### ISSUE TITLE
It's not working anymore on Chrome

### ISSUE THREAD

**Issue by federicocappellotto97** [kenwheeler/slick#4056]
I'm having problem with slick slider on chrome, on mac Os Big sur. Last week it worked well, now sliders are not showing. Instead with Safari or Firefox it's working as always

**Comment by runnickrun** [comment]
Even the homepage of slider itself doesn't work.

**Comment by federicocappellotto97** [comment]
Same, if i disable uBlock it works well..

**Comment by grialexandru** [comment]
go to uBlock extension -> dashboard -> filter list 

1. purge all cache
2. update filter list

it should work after this

**Comment by wgnahs** [comment]
Can confirm, it seems that the problem has been solved!

**Comment by federicocappellotto97** [comment]
Thanks, not it's working!

**Comment by clementmarguet** [comment]
Same problem here, is there any solution server side ?

**Comment by wgnahs** [comment]
> Same problem here, is there any solution server side ?

Have you tried to purge the cache in your uBlock extension? Haven't had the issue anymore since @grialexandru's reply.

**Comment by clementmarguet** [comment]
It's ok for me with the above solution, but i have Slick running on a client website, i cannot tell his users with ublock to purge their cache.

**Comment by runnickrun** [comment]
Indeed, we can't tell everyone to clear their cache and add to the filter. Is this issue on slick side or ublock?

**Comment by darthnorman** [comment]
I think this was an issue in one of the ublock filter lists. I don't know why the updated lists aren't rolled out to everyone.

**Comment by clementmarguet** [comment]
Yep, i don't have the bug anymore.

**Comment by ouw-devs** [comment]
Honestly they should put that filter rule back. The world would be better off.
